src/data-dictionary/events/ContainerSample/memoryUsageBytes.md

```diff
@@ -6,4 +6,6 @@ events:
   - ContainerSample
 ---
 
-The total memory the container is using.
\ No newline at end of file
+The amount of memory the container is using. This memory doesn't include swap.
+
+**Breaking change**: This metric includes memory (including swap) on versions v1.5.0 or higher of `nri-docker`.
```

Comment:
Users probably do not know which version of `nri-docker` they have since it is bundled with the agent.

```suggestion
The amount of memory the container is using. This memory doesn't include swap.

**Breaking change**: This metric includes memory (including swap) on versions v1.5.0 or higher of `nri-docker`.
```
```suggestion
The amount of memory the container is using. This memory does not include include swap usage.

**Breaking change**: This metric used to include swap memory in `nri-docker` v1.5.0 and earlier (Infrastructure Agent `1.16.x` and earlier). The amount of used memory including swap is now reported as `memorySwapUsageBytes`.
```